test/mocks/Strategy.ts

```diff
@@ -56,7 +56,8 @@ export async function setupMockStrategy(
             // The logic below runs for old strategies that don't present a major harvesting issue. 
             log(`old strategy address:, ${oldStrategy_addr}`);
             const oldStrategy = new ethers.Contract(oldStrategy_addr, stratABI, governanceSigner);
-            const harvest = await oldStrategy.connect(governanceSigner).harvest();
+            // const harvest = await oldStrategy.connect(governanceSigner).harvest();

```

Comment:
Is this change correct for all existing strategy tests?

Reply:
nope

Reply:
instead of commenting out, is it possible to condition on the strategy name? 
is this for a specific strategy or all bankerjoe/some platforms strategy?

Reply:
looks like none of the bankerJoe strats have harvest. please add this condition to ensure the contract has the harvest function.
```
            ...
            if (oldStrategy.functions['harvest']) {
               const harvest = await oldStrategy.connect(governanceSigner).harvest();
               const tx_harvest = await harvest.wait(1);
               if (!tx_harvest.status) {
                   console.error(`Error harvesting the old strategy for: ${contract_name}`);
                   return Strategy;
                }
                log(`Harvested the old strategy for: ${contract_name}`);
            }
            ...
```